config/application.yml

```diff
@@ -0,0 +1,16 @@
+test_key: "123qwe"
```

Comment:
This file should definitely be in `gitignore` - everyone can see all your secret keys.